moveit_ros/perception/pointcloud_octomap_updater/include/moveit/pointcloud_octomap_updater/pointcloud_octomap_updater.h

```diff
@@ -47,11 +47,19 @@ DIAGNOSTIC_POP
 #include <sensor_msgs/PointCloud2.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_updater.h>
 #include <moveit/point_containment_filter/shape_mask.h>
+#include <moveit_msgs/UpdatePointcloudOctomap.h>
 
 #include <memory>
+#include <mutex>
 
 namespace occupancy_map_monitor
 {
+enum class UpdateMethod
```

Comment:
Please add a better description of the two update modes. What's the difference between them?